src/_includes/head.html

```diff
@@ -4,6 +4,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   {%- seo -%}
   <link rel="stylesheet" href="{{ "/assets/css/style.css" | relative_url }}">
+  <link rel="stylesheet" href="{{ "/assets/css/slideshow.css" | relative_url }}">
```

Comment:
let's put this just in the page that needs it, rather than on every page

Reply:
yes! i assume you mean include it in head via conditional template? https://idratherbewriting.com/documentation-theme-jekyll/mydoc_conditional_logic.html

Reply:
done!